server/chat-plugins/room-faqs.ts

```diff
@@ -3,28 +3,66 @@ import {FS, Utils} from '../../lib';
 export const ROOMFAQ_FILE = 'config/chat-plugins/faqs.json';
 const MAX_ROOMFAQ_LENGTH = 8192;
 
-export const roomFaqs: {[k: string]: {[k: string]: string}} = JSON.parse(FS(ROOMFAQ_FILE).readIfExistsSync() || "{}");
+export const roomFaqs: {[k: string]: {[k: string]: RoomFAQ}} = (() => {
+	const data = JSON.parse(FS(ROOMFAQ_FILE).readIfExistsSync() || "{}");
+	let save = false;
+	for (const k in data) {
+		for (const name in data[k]) {
+			if (typeof data[k][name] === 'string') {
+				data[k][name] = convertFaq(data[k][name]);
+				save = true;
+			}
+		}
+	}
+	if (save) saveRoomFaqs();
+	return data;
+})();
+
+interface RoomFAQ {
```

Comment:
I'd prefer something like
```ts
type RoomFAQ = {alias: string} | {source: string, isHTML?: boolean};
``` 
but it's not a big deal to have the alias in the `source` I guess.